client/src/components/DoctorCard.js

```diff
@@ -59,7 +34,9 @@ const DoctorCard = () => {
       title: "Card Title 2",
       description: ["Description for the second card.", "Some more text."],
       actions: [
-        <Button key="share">Check availability</Button>,
+        <Button key="share" onClick={showModal}>
+          Check availability2
```

Comment:
Halka sa typo, I will change

Reply:
Or is it just the name for the second button?

Reply:
Checking the second button at that time. Will change it in the next PR